test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/CollectionTest.scala

```diff
@@ -320,6 +332,10 @@ trait CollectionTest extends IterableTest {
         expected.contains(result))
   }
 
+  @Test def constructorNullThrowsNullPointerException(): Unit = {
```

Comment:
And this one should move down to `ArrayListTest`, since it specifically instantiates an `ArrayList` (and it has to).